src/lib/accessibility/Behaviors/List/SelectableListItemBehavior.ts

```diff
@@ -11,6 +13,14 @@ const SelectableListItemBehavior: (props: any) => IAccessibilityDefinition = (pr
     root: {
       role: 'option',
       'aria-selected': !!props['active'],
+      [IS_FOCUSABLE_ATTRIBUTE]: true,
+    },
+  },
+  keyActions: {
+    root: {
+      performClick: {
+        keyCombinations: [{ keyCode: keyboardKey.Enter }, { keyCode: keyboardKey.Spacebar }],
```

Comment:
Spacebar scrolls page down, so we should add some clever way of preventing default of certain key actions. There is already the same problem with `Menu` (you can try in docsite). What do you think?

Reply:
yes - the user will need to do preventDefault for now, we do not have a generic solution yet.